cluster-monitoring/templates/prometheusrule.yaml

```diff
@@ -29,3 +30,36 @@ spec:
       annotations:
         description: '{{`{{$labels.instance}}`}} with calico-node pod {{`{{$labels.pod}}`}} has been having dataplane failures'
         summary: 'Instance {{`{{$labels.instance}}`}} - Dataplane failures'
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: "{{ .Release.Name }}-node-custom"
+  namespace: "{{ .Release.Namespace }}"
+  labels:
+    app: prometheus
+    prometheus: k8s-monitor
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "k8s-monitor"
+    heritage: "{{ .Release.Service }}"
+    role: alert-rules
+spec:
+  groups:
+  - name: node-custom.rules
+    rules:
+    - alert: MemoryUsageHigh
+      expr: (sum(node_memory_MemTotal) - sum(node_memory_MemFree + node_memory_Buffers + node_memory_Cached) ) / sum(node_memory_MemTotal) * 100 >= 90
```

Comment:
this will give you a total for the full cluster, but not a per-node check like you wanted.

something like `(sum by (instance) (node_memory_MemTotal) - sum by (instance) (node_memory_MemFree + node_memory_Buffers + node_memory_Cached) ) / sum by (instance) (node_memory_MemTotal) * 100 >= 90` will work better